src/quo2/components/account_selector.cljs

```diff
@@ -0,0 +1,80 @@
+                             :label-text     colors/neutral-50}
+           :transparent     {:bg      colors/neutral-80-opa-5
+                             :account-text     colors/black
+                             :label-text     colors/neutral-50}}
+
+   :dark  {:default         {:bg      colors/neutral-80-opa-80
+                             :account-text    colors/white
+                             :label-text     colors/neutral-40}
+           :transparent     {:bg      colors/white-opa-5
+                             :account-text    colors/white
+                             :label-text     colors/neutral-40}}})
+
+;we prefer using temp uri for assets so that un-neccessary assets do not become a part of release bundle
+(def temp-fire-icon-uri "https://s3-alpha-sig.figma.com/img/58f0/9735/781a2f7ece5e5c0fde2cb7e9d2c22c77?Expires=1661731200&Signature=f5hvtb9Wz1Zwd4-dft9gOyBP25rR1wrVFtHyCQh9NCslCBZ7JJzex7qYdYZn2Qvf6ep50HIdkO1oh6wVBqeds1BrX0T4ojSzM8HA0CQkXlVveEwEyjcqb~XPx3m2zMFEcaoUX87Ho3I6ZHK477WwoHgKX9JwfpfoNZBSqVT1TiTiTPpNdv4AEY-5TJcuZk7mS1-O1eCuEoQA-VTWX3UXExsZ9wip-rnrp2o8a7dx1run9jHSNqsKfEMgK2Mc3zjphzW34asgrxCf1plRv3ZD12ht~pzsuph8FeeBb9HFLtumbUKbDI7L6sffEFGY6msxUBr0ret0akFlk1x00gnwCA__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA")
+
+(defn account-container-row [background-color]
+  {:height 40
+   :flex-direction :row
+   :align-items :center
+   :background-color background-color
+   :border-radius 12
+   :width 355})
+
+(def account-emoji
+  {:height 16
+   :width 16})
+
+(def account-emoji-container
+  {:background-color colors/purple-50
+   :height 32
+   :width 32
+   :justify-content :center
+   :align-items :center
+   :border-radius 10
+   :margin-left 4
+   :margin-right 8})
+
+(defn account-text-style [account-text-color]
+  {:color account-text-color
+   :font-weight :500
+   :font-size 15})
+
+(defn account-label [label-text-color]
+  {:font-size 13
+   :font-weight :500
+   :margin-bottom 6
+   :color label-text-color})
+
+(defn get-color-by-type [type key]
+  (get-in themes [(theme/get-theme) type key]))
+
+(defn account-selector
+  "[account-selector opts]
+   opts
+   {:show-label?       true/false            ;; hide or show the label
+    :transparent?      true/false            ;; implement transparent background styles
+    :style             style                 ;; any other styling can be passed
+    :label-text        \"Label\"             ;; content to show where the label should be shown
+    :account-text      \"My Savings\"        ;; content in place of account name
+   }"
+  [{:keys [show-label? account-text transparent? label-text style]}]
+  (let [background-color (get-color-by-type (cond transparent? :transparent :else :default) :bg)
+        account-text-color (get-color-by-type (cond transparent? :transparent :else :default) :account-text)
+        label-text-color (get-color-by-type (cond transparent? :transparent :else :default) :label-text)]
+    [rn/view (cond (> (count style) 0) {:style style})
```

Comment:
you don't need this condition

Reply:
why cond and not `when` ?

Reply:
also what is `(count style)` ?

Reply:
> you don't need this condition

I wanted to first check if `style` parameter was provided and only set components `style` when it was passed, I'll refactor

Reply:
`{:style nil}` will work fine

Reply:
testing it locally, will push these changes in a bit